website/docs/r/vapp.html.markdown

```diff
@@ -75,3 +131,5 @@ The following arguments are supported:
 * `metadata` - (Optional) Key value map of metadata to assign to this vApp
 * `ovf` - (Optional) Key value map of ovf parameters to assign to VM product section
 * `power_on` - (Optional) A boolean value stating if this vApp should be powered on. Default to `true`
```

Comment:
Make sure the defaults are listed in the docs. It looks like there are some that are in the schema, but not in the docs. For the most part it looks like pre-2.0. From the ones I've spot checked, it looks like all the 2.0 changes are documented, but wanted to point it out to try and get everything consistent.